snakebite/client.py

```diff
@@ -393,6 +394,60 @@ def _handle_rename(self, path, node, dst):
         response = self.service.rename(request)
         return {"path": path, "result": response.result}
 
+
+    def rename_dont_move(self, path, dst, overwriteDest=False):
```

Comment:
Somehow I have something against the name - what do you think about:
 * rename2
 * rename_to
?